repair_follow_lot_location/models/repair_order.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class RepairOrder(models.Model):
+
+    _inherit = "repair.order"
+
+    follow_lot_location = fields.Boolean(
```

Comment:
IMO it should be a option into the settings for the current company :thinking: For an end user responsible of repair order, I've the feeling that this configuration is too specific...

Reply:
In our use case, this information is set on the RMA, with the default taken from the RMA reason. I’m writing a glue module, rma_repair_follow_lot_location, for this

Reply:
That’s why, IMO, this option should have its default set on the RMA reason, aligned with the auto_create_lot setting
people not using the RMA stack for repairs should manage this option manually, since they create repair orders from scratch any way